### PR TITLE
added Button component

### LINKED ISSUE #1: Create Button component
* [x] Depends on #5

[Zeplin](https://app.zeplin.io/project/5e76a2aae68a5e6f93af1053/styleguide/components?cseid=5e778ced90d9966e42dabfae)

image

### PR DESCRIPTION
**Pull request by gabrielperales**
closes #1

### CODE REVIEW

File: src/components/Button/index.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+};
+
+const sizeToTextSize = (size?: ButtonSize) => size;
+
+const ButtonWrapper = styled.div`
+  display: inline-block;
+  background-color: ${({ variant }: Partial<Props>) =>
+    variantToBgColor(variant)};
+  box-shadow: ${({ variant }: Partial<Props>) => variantToBoxShadow(variant)};
+  padding: ${({ size }: Partial<Props>) => sizeToPadding(size)};
```
Comment (gabrielperales) [inline src/components/Button/index.tsx:49]: ops, I forgot to add `cursor: pointer;` 😅

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title data-react-helmet="true"></title>
+    <script src="/socket.io/socket.io.js"></script>
+  </head>
+  <body>
+    <div id="___gatsby"></div>
+    <script src="/commons.js"></script>
+  </body>
+</html>
```
Comment (breadadams) [inline public/index.html:17]: What's this file for?

**Review by breadadams — APPROVED** [review]

File: public/index.html (hunk 2 above)
Comment (gabrielperales) [reply public/index.html:17]: don't know... Did I push that? it must be an auto-generated file

Comment (breadadams) [reply public/index.html:17]: I don’t think so, it says Gatsby, and you don’t creste an `index.html` with next

**Pull request merged**